docs/resources.mdx

```diff
@@ -53,3 +53,29 @@ route: /resources
 ## Misc
 
 - [Cryptotux](https://cryptotux.org/)
+
```

Comment:
Can you wrap this in a new `<ul class="UnorderedList-kkVuiO jvAwzV">`? 

This is now just a dangling `<li>` element, since markdown grabs all concurrent `-` and throws then into a `<ul>`

Reply:
Yup yup, wrapping it in the appropriate ul made things easier. Also made the kitty slightly smaller. Thanks for the input!